### ISSUE TITLE
Next priority list of updates & fixes before new website going live

### ISSUE THREAD

**Issue by lulen11** [aimementoring/website#90]
Hey team, 

Creating this as as an epic with a checklist of all the smaller issues listed here (**please edit and add to this** as you please). 

_I think best thing would be for us to first add the item to the checklist here. Then go and create an issue or directly open a PR that would tackle the checklist item (and would provide a place for us to have specific conversation directly related to said issue). Then please link the issue/PR you created to the checklist item in here, as well as tagging the main person responsible for each item._ 

### First priority:
- [x] Update node version - https://github.com/aimementoring/website/issues/91 @kbardi 
- [x] Link blueprint version - https://github.com/aimementoring/website/issues/92 @kbardi 
- [x] Fix why the images aren't displaying - https://github.com/aimementoring/website/issues/93 @kbardi 
- [x] Get SEO functioning - https://github.com/aimementoring/website/issues/88 @kbardi  
- [x] Be a mentor temporary updates - https://github.com/aimementoring/website/issues/94 @lulen11 
- [x] Update newsletter component with mailchimp form - https://github.com/aimementoring/website/issues/95 @lulen11 
- [x] Update intercom widget - https://github.com/aimementoring/website/issues/102 @rin 
- [x] Add beta message. @charliemckenzie @rin if https://github.com/aimementoring/blueprint/issues/132 needs longer then this week then @lulen11 maybe you just hardcode something simple for now. 
- [x] Check/fix donate page. May be broken because v5 domain is blocked by Raisly @kbardi 
- [x] /the-mentor broken. Proposed long term solution was discussed in #32. My short term solution [here](https://www.loom.com/share/40037f7728364b6abd3d66578b13d8e9) @lulen11 @CarolinaPd   
- [x] Footer: Remove Become a Partner link / make it activate Intercom if easy @lulen11 
- [x] Footer: "Shop Apparel" isn't linked to the shop @lulen11 
- [x] Footer: Hide FAQs link until they can be re-written #103 @lulen11
- [x] Fonts - fix font-stack if fonts don't load @lulen11 

### Second priority:
- [x] Update donate page link to widget - https://github.com/aimementoring/website/issues/97 @bronz3beard 
- [x] Restyle dropdown nav items @lulen11 

### Nice to have:
- [ ] Visual fixes on Stories section - https://github.com/aimementoring/website/issues/96 @lulen11 
- [x] Vertically centre nav items ([it’s most pronounced when sticky](https://www.dropbox.com/s/m4cfvfxry2my642/Screenshot%202020-01-21%2010.35.57.png?dl=0)) 

-----
_Side note: I think I should have created this as a “release” not an “epic” but too late now. I'm aware... My bad._
